docs/design/storage-node-automatic-updater.md

```diff
@@ -0,0 +1,95 @@
+# Storage Node Automatic Updater
+
+## Abstract
+
+Automatic Updater is a process separate from the Storage Node that automatically downloads the latest Storage Node binary to replace the current Storage Node binary.
+
+## Background
+
+As more Storage Node Operators join the network we must ensure these nodes have a mechanism to automatically keep their nodes software up to date.
+If a Storage Node Operator fails to keep their node up to date with the minimum version required by satellites they will no longer be selected for upload or download requests.s
+Currently we are using Docker for automatic updates but we are migrating away from docker so we need to write out own automatic update utility for Storage Nodes.
+
+## Design
+
+* Contacts Version Server to determine if Storage Node needs to be updated.
+* Downloads latest Storage Node binary.
+* Validate binary signature.
+* Stop the Windows Service for Storage Node
```

Comment:
We are designing a platform-independent autoupdater, right? In such case, let's just say "Stop the Storage Node process". On Windows, this would indeed mean to stop the Windows Service.